### PR TITLE
added integration tests

### PR DESCRIPTION
**Pull request by abolla22**

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+build_parameters:
+  - "--suppress-variables"
+  - "--skip-existing"
+  - "--error-overlinking"
+  - "--error-overdepending"
+  - "-c distro-tooling"
```
Comment (dkomisar) [inline abs.yaml:6]:
```suggestion
  - "-c distro-tooling"
 
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -50,10 +50,17 @@ test:
     - hypothesis >=3.58
     - pytest >=6.0
     - pytest-xdist
+    - package-testing
   imports:
     - pandas
+  source_files:
+    - package-testing/anaconda_package_tests
```
Comment (dbast) [inline recipe/meta.yaml:57]: That does not work and is not required here
```suggestion
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -50,10 +50,17 @@ test:
...
   commands:
     - pip check
+    - pytest
+    - pytest -s anaconda_package_tests
+
+
```
Comment (dbast) [inline recipe/meta.yaml:63]: Requires specifying the path where pytest can find the packaged tests, see also https://github.com/anaconda-distribution/package-testing/blob/main/recipe/meta.yaml#L34
```suggestion
    - pytest -s -vv ${CONDA_PREFIX}/lib/python${PY_VER}/site-packages/anaconda_package_tests
```

Comment (abolla22) [reply recipe/meta.yaml:63]: Sure , since the folder is added . thought providing the directory name would suffice. Now will add the full path , Thanks.

**Pull request closed without merging**